oneanddone/tasks/views.py

```diff
@@ -149,10 +149,10 @@ def get_context_data(self, *args, **kwargs):
         return ctx
 
     def form_valid(self, form):
-        form.save(self.request.user)
-
+        task = form.save(self.request.user)
+        
```

Comment:
This blank line contains whitespace. Your `pep8` or `flake8` checker should have caught that. Do you have a pre-commit hook set up to check for `pep8` errors?